exercises/dot-dsl/Example.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+public class Node : Element
+{
+    public Node(string name)
+    {
+        Name = name;
+    }
+
+    public string Name { get; }
+
+    public override bool Equals(object obj)
+    {
+        if (obj == null || GetType() != obj.GetType())
```

Comment:
Assuming you're using C# 6 you can simplify the if to:
`if (GetType() != obj?.GetType())`

Also applies to other classes where you've overridden `Equals`.